cmd/docker/main.go

```diff
@@ -46,7 +46,7 @@ type CliOptions struct {
 	proto         string
 }
 
-const productName = "vSphere Integrated Containers"
+const productName = "vSphere Integrated Containers 0.3"
```

Comment:
Let's create a version file or a build tag for this and either read or inject into at compile time